app/views/listings/_listing_actions.haml

```diff
@@ -170,6 +171,11 @@
         %a.icon-with-text-container{href: edit_listing_path(@listing)}
           = icon_tag("edit", ["icon-part"])
           .text-part= t("listings.edit_links.edit_listing")
+        - if FeatureFlagHelper.feature_enabled?(:manage_availability)
+          - if show_manage_availability
+            = react_component("ManageAvailabilityApp", props: manage_availability_props, prerender: false)
+          - else
+            %p availability not enabled for listing
```

Comment:
I left this here for testing purposes - I guess this else option is not necessary otherwise?